lib/CL/pocl_binary.c

```diff
@@ -322,12 +318,12 @@ pocl_binary_get_kernel_count (cl_program program, unsigned device_i)
 /***********************************************************/
 
 /* serializes a single file. */
```

Comment:
WHile at this, can you expand on the comment with the arguments described?

Reply:
Addressed.